snmp/datadog_checks/snmp/data/default_profiles/dlink-dgs-switch.yaml

```diff
@@ -0,0 +1,169 @@
+extends:
+  - dlink.yaml
+
+sysobjectid:
+  - 1.3.6.1.4.1.171.10.137.*
+  - 1.3.6.1.4.1.171.10.137.1.1
+
+metrics:
+  - MIB: DLINKSW-ENTITY-EXT-MIB
+    table:
+      name: dEntityExtCpuUtilTable
+      OID: 1.3.6.1.4.1.171.14.5.1.7
+    symbols:
+      - name: cpu.usage
+        OID: 1.3.6.1.4.1.171.14.5.1.7.1.3 # dEntityExtCpuUtilOneMinute
+    metric_tags:
+      - column:
```

Comment:
Ideally we should tag cpu.usage metric with `cpu` tag, but it seems that the `dEntityExtCpuUtilTable` have multiple indexes, so not easy to tag.

So, not sure what value to give to `cpu` tag  in this case.

```
        INDEX   { 
            dEntityExtCpuUtilUnitId,
            dEntityExtCpuUtilCpuID
        }
```

Reply:
I've added `dEntityExtCpuUtilUnitId ` which is a larger category (you have multiple cpu id for each unit id), this way graphing by `cpu` would make more sense with showing the cpu per unit (aggregated by cpu id), rather than showing the cpu per ID (aggregated by unit id).

For readability, I'm duplicating the tag though